packages/manager/apps/ips/src/pages/listing/components/DatagridCells/IpAntiDdos/IpAntiDdos.tsx

```diff
@@ -12,11 +16,15 @@ export type IpAntiDdosProps = {
 /**
  * Component to display the cell content for Anti DDOS.
  * If ip is not /32 (isGroup = true) or not ipv4, we display nothing.
- * If ip mitigation is Permanent we display permanent if no mitigation or not permanent, we display automatic
+ * If ip mitigation is Permanent we display permanent
+ * If ip mitigation is Auto, we display In Action
+ * If ip mitigation has no mitigation, we display automatic
+ * If mitigation state is not "ok", we display pending
  * @param ip the ip with mask
  * @returns React component
  */
 export const IpAntiDdos = ({ ip }: IpAntiDdosProps) => {
+  const id = `antiddos-${ip.replace(/\/|\./g, '')}`;
```

Comment:
```suggestion
  const id = `antiddos-${ip.replace(/\/|\./g, '_')}`;
```

in case we have things like 10.10.1.0 and 10.1.0.10 for example ?